### ISSUE TITLE
Badfilter created from logger does not work

### ISSUE THREAD

**Issue by gwarser** [uBlockOrigin/uBlock-issues#732]
### Prerequisites

- [x] I verified that this is not a filter issue
    - Filter issues MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues)
    
- [x] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uBlockOrigin](https://old.reddit.com/r/uBlockOrigin/)
    
- [x] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
- I tried to reproduce the issue when...
    - [x] uBlock Origin is the only extension
    - [x] uBlock Origin with default lists/settings
    - [x] using a new, unmodified browser profile
- [x] I am running the latest version of uBlock Origin
- [x] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### Description

Filters manually created from _Static filter_ creator in _the logger_, by manually appending `,badfilter` option don't work until re-applied from _My filters_.

### A specific URL where the issue occurs

[A specific URL is MANDATORY for issue happening on a web page, even if it happens "everywhere"]

### Steps to Reproduce

https://github.com/orgs/uBlockOrigin/teams/ublock-issues-volunteers/discussions/139

- open https://www.msn.com
- open logger and switch to "All"
- click in msn search box and press enter
  - bing popup should open
- click on `@@||bing.com/$popup,domain=msn.com` logger line
- copy exception filter if you don't have it already in clipboard
- switch to Static filter creator
- paste filter overwriting already present content
- append `,badfilter`
- "Create" filter
- close bing tab and retry searching (click input box -> enter)
- no change in behavior, bing popup still open

Filter was added to My filters.

It can be "activated" by modifying something in My filters and applying changes.

***

https://github.com/orgs/uBlockOrigin/teams/ublock-issues-volunteers/discussions/89

- choose an adserver, for example `http://serverflox.online/` which is blocked in ublock filters by:
`||serverflox.online^`
- go to `http://serverflox.online/`
- strict blocking page => enable temporarily
- in the logger click the blocked request `||serverflox.online^`
- create /edit a filter (in the logger) `||serverflox.online^$badfilter`
- go again to `http://serverflox.online/` => the `badfilter` is ignored even if in `my filters` (still seeing `||serverflox.online^` in the logger)

- edit `my filters` (add a comment, delete a character in a comment) 
- go again to `http://serverflox.online/` => now the `badfilter` is active (no blocked request in the logger)

### Expected behavior:

[What you expected to happen]

### Actual behavior:

[What actually happened]

### Your environment

* uBlock Origin version: 1.22.3b16
* Browser Name and version: Firefox 71
* Operating System and version: Manjaro KDE

**Comment by krystian3w** [comment]
Also `uBO logger` show badfilter in old detected blocked network request if we add to My Filters Panel.

![obraz](https://user-images.githubusercontent.com/35370833/65820401-a36e6100-e228-11e9-8325-a0360880ef56.png) |
--- |

![obraz](https://user-images.githubusercontent.com/35370833/65820416-dadd0d80-e228-11e9-8c93-1280aef99f4a.png) |
--- |

such detection misleads users...

**Comment by gwarser** [comment]
@krystian3w it's different, why not create new issue?

**Comment by krystian3w** [comment]
To many is opened.

**Comment by mapx-** [comment]
see
https://github.com/uBlockOrigin/uBlock-issues/issues/2507

**Comment by gwarser** [comment]
@mapx- `badfilter` still does not work.

**Comment by mapx-** [comment]
right, 

I can reproduce it:

- go to msn.com
- see in the logger `||msn.com/onecollector/` filter
- click the row, static filter, add `||msn.com/onecollector/$badfilter`
- the filter is "seen" by uBO only after a change in "my filters" (newline, comment modification etc)

@gorhill
